internal/cmds/list/main.go

```diff
@@ -342,6 +344,40 @@ func listControls(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
+func listModulePars(cmd *cobra.Command, args []string) error {
+	for _, info := range infos {
+		if info.Err != nil {
+			return info.Err
+		}
+		ast.Inspect(info.Module, func(n ast.Node) bool {
+			if n == nil {
+				return false
+			}
+			switch n := n.(type) {
+			case *ast.ValueDecl:
+				if n.Kind.Kind == token.MODULEPAR || n.Kind.Kind == token.ILLEGAL {
```

Comment:
why is it necessary to compare ValueDecl with token.ILLEGAL?

Reply:
The first token of a value declaration (`ValueDecl.Kind`) usually denotes the kind: `var`, `const`, `modulepar`, ... For the deprecated module parameter group syntax (`modulepar { int x }`) however this denoting token does not exist (`token.ILLEGAL`).

Current AST implementation does not provide good alternatives, yet.